src/flags/update-block-height.flag.ts

```diff
@@ -0,0 +1,17 @@
+import {Flags} from '@oclif/core'
+
+export enum BlockHeightOptions {
+  API = 'api',
+  FILE = 'file',
+  DISABLE = 'disable',
+}
+
+export const blockHeightFlag = {
+  updateBlockHeight: Flags.string({
+    aliases: ['update-block-height'],
+    description: 'Space separated list of networks to use',
```

Comment:
The description should be: "Define how to save the last block that was processed."